### ISSUE TITLE
Mass Definition

### ISSUE THREAD

**Issue by steven-murray** [steven-murray/hmf#6]
Changing mass definition (delta_h) should change all mass functions, not just those with explicit dependence.

**Comment by steven-murray** [comment]
This is address in branch https://github.com/steven-murray/hmf/commits/feature/mass-definition. Will close this when it is pulled into develop.

**Comment by safonova** [comment]
Would like to note that this is still an issue. Of the models we have checked, we have found that the results from MST, Angulo, Courtin, Jenkins, and Watson (FoF Universal) stay the same regardless of how delta halo is defined. Only the halo mass function values from Tinker 2008 vary. Thanks.

**Comment by steven-murray** [comment]
Thanks for the details! If you require these changes ASAP, please update hmf by doing:

    pip install git+git://github.com/steven-murray/hmf@feature/mass-definition

Also, see https://github.com/steven-murray/hmf/blob/feature/mass-definition/docs/examples/change_mass_definition.ipynb for details on how to use the new mass-definition interface.

Please report any bugs/issues that you have! This is due for integration into master branch soon, and it would be excellent if any bugs are ironed out first!

Note that the above instructions will be invalid once the branch is merged (for future reference).

**Comment by steven-murray** [comment]
This has been merged into develop.
